_posts/en/posts/2017-09-01-release-0.15.0.md

```diff
@@ -0,0 +1,130 @@
+
+## Performance improvements
+
+As part of the continuing effort to make full nodes available to as many users as possible even as the block chain continues to grow in size and complexity, Bitcoin Core 0.15.0 includes several significant performance improvements.
+
+- **30% to 40% faster block validation and 10% to 20% less memory used** on tests of Initial Block Download (IBD), with far fewer writes to disk.  This is the result of simplifying the format of the the chainstate database that tracks each spendable group of bitcoins and what information the owner of those bitcoins needs to provide in order to spend them.
+
+- **40% to 50% faster validation of blocks consisting of previously-seen transactions** as the result of repeating fewer validation steps when a previously-verified mempool transaction is later received in a block.
+
+- **Moderate performance gains on some platforms** as the result of using hardware acceleration for some operations, such as support on modern computer processors for single-SHA256 hashing which results in about a 50% improvement each time one of Bitcoin's double-SHA256 hashes needs to be computed.  This mainly benefits users of 64-bit Intel and AMD processors produced in 2008 or later.
+
+More information on each of these improvements may be found in the [release notes][].  
+
+## The future: P2SH-wrapped segwit addresses
+
+As final preparations are being made to release Bitcoin Core 0.15.0, segregated witness has activated on the Bitcoin network and is now ready to use.
+
+Bitcoin Core has supported creating segwit addresses since 0.13.0, but this support was designed for testing has only been available to expert users---we were waiting to see if segwit was adopted before adding segwit support to the regular user interfaces, both graphical and RPC.
+
+The timing of segwit lock in and activation meant that we had to choose between either delaying the planned release of 0.15.0 and all its features described above or shipping 0.15.0 without a user interface defaulting to segwit.
+
+We decided to take the later option, but we're also not going to wait the normal six months before the next major update.  Instead, our next release will be a feature release that generates segwit-compatible addresses by default.  This will be made available as soon as it has been written and thoroughly tested.
+
+For those of you interested in technical details, our plan is to use P2SH-wrapped segwit addresses that are compatible with nearly all other wallets on the network.  Bech32 native segwit addresses will either be implemented in a subsequent release (assuming no problems arise) or will only be available through expert options for now.
```

Comment:
I believe the plan is to support Bech32 native segwit addresses for sending only until the rest of the community updates.